README.md

```diff
@@ -1,219 +1,121 @@
+forked from the [legacy Keycloak Operator](https://github.com/keycloak/keycloak-operator) and stripped off of any functionality
+related to Keycloak Deployment as it was designed for the WildFly distribution of Keycloak and is not compatible with the
+new Quarkus distribution. The Realm Operator is meant as a **temporary workaround** to compensate for the missing
+User and Client CR in the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator) and to work
+as a side-car for it.
+
+## Motivation
+
+To provide the best experience, the new Keycloak Operator will use a new approach to manage Keycloak resources, such as Realms, Clients and Users. This approach will leverage the [new storage architecture](https://www.keycloak.org/2022/07/storage-map.html) and future immutability options, making the CRs the declarative single source of truth. In comparison to the [legacy Operator](https://github.com/keycloak/keycloak-operator), this will bring high robustness, reliability and predictability to the whole solution.
+
+Before we would consider the new Keycloak Operator ready for leveraging CRs, we expect completing several features including but not
+limited to:
+
+* File store (expected in Keycloak 20) to persist data in a file instead of DB.
+* Read-only administration REST API, UI Console and other interfaces. This is required for the new immutability concept
+  which will be used to ensure any data coming from the CRs (and subsequently from the file store) are read-only from
+  all interfaces.
+
+All of this is critical to proper CRs implementation, hence the new Keycloak Operator is currently missing the CRDs for managing
+Keycloak resources. The missing CRDs will be added once Keycloak has the necessary support for it, which is currently
+expected in Keycloak 21.
+
+There are two alternatives to mitigate the current lack of CRs in the new Keycloak Operator:
+* Using the Realm Import CR from the new Keycloak Operator without using the Realm Operator
+* Using the Realm Operator for managing Realm resources and new Keycloak Operator for deploying Keycloak
+
+For details, please see the following chapters.
+
+## Using the Realm Import CR and  the new Keycloak Operator
+
+As the name suggests, the Realm Import CR is meant only for declarative Realm creation (no updates or deletion is supported), very much like the [Realm CR in the legacy Operator](https://github.com/keycloak/keycloak-operator/blob/main/deploy/crds/keycloak.org_keycloakrealms_crd.yaml) which was used the same way. The main advantage and difference of the Realm Import CR are that it contains the full Realm representation (incl. all sub-resources like Clients, Users, etc.) unlike the old Realm CR which included only selected fields.
+
+To learn more about the Realm Import CR, see the [documentation](https://www.keycloak.org/operator/realm-import).
+
+## Using the Realm Operator
```

Comment:
I'm a bit confused here. Why mention the "Realm Operator" at all? I'd rather just make the new recommended thing the new Operator + realm manger operator, and leave the old operator completely out (as we won't do any updates to it after KC 20 is out).